addons/website/static/src/snippets/s_popup/000.js

```diff
@@ -202,6 +203,65 @@ const PopupWidget = publicWidget.Widget.extend({
 
 publicWidget.registry.popup = PopupWidget;
 
+publicWidget.registry.cookies_bar = PopupWidget.extend({
```

Comment:
This code is using a separate widget that extends the element's widget to target the same element (with a different selector...), so for example, the same `PopupWidget.start()` code will be called twice on start :grimacing: Is there something that prevents extending `publicWidget.Widget` :thinking: ? (since the "Popup" behavior is already handled by the `PopupWidget` code).

Reply:
After a quick look in the code, the `cookies_bar` widget calls the `_onHideModal()` from the `PopupWidget`. So I guess that's why.